pallets/asset/src/lib.rs

```diff
@@ -338,6 +338,17 @@ decl_storage! {
 
         /// Ticker registration details on Polymath Classic / Ethereum.
         pub ClassicTickers get(fn classic_ticker_registration): map hasher(blake2_128_concat) Ticker => Option<ClassicTickerRegistration>;
+        /// Balances get stored on the basis of the `ScopeId`.
+        /// Right now it is only helpful for the UI purposes but in future it can be used to do miracles on-chain.
+        /// (ScopeId, IdentityId) => Balance.
+        pub BalanceOfAtScope get(fn balance_of_at_scope): double_map hasher(identity) ScopeId, hasher(identity) IdentityId => T::Balance;
```

Comment:
What's the point of this mapping? Shouldn't `Ticker` be part of the mapping as well? Even then, what's the intended use case here?

Reply:
This mapping is only for the UI right now which will allow UI to provide a list of Identities under one `ScopeId`. We can add the ticker in the mapping as well if we want to curate on the basis of the Ticker. Right now we don't have explicit use of this mapping. If we want we can remove it.

Reply:
I think it's best for the UI to use events to curate this list. If we want this to be onchain, it should be `(scopeid, identity) -> boolean`. Having it point to an arbitrary balance is just wrong.

Reply:
How it is arbitrary? FYI -`ScopeId` is unique for every ticker & InvestorUid.